tools/Wasm.mk

```diff
@@ -54,10 +54,16 @@ CFLAGS_STRIP += $(ARCHCPUFLAGS) $(ARCHCFLAGS) $(ARCHINCLUDES) $(ARCHDEFINES) $(A
 WCFLAGS += $(filter-out $(CFLAGS_STRIP),$(CFLAGS))
 WCFLAGS += --sysroot=$(WSYSROOT) -nostdlib -D__NuttX__
 
+# Keep optimization and lto flags
+
+WCFLAGS += $(filter -O%, $(ARCHOPTIMIZATION))
```

Comment:
should we decouple native compiler option from wasm compiler?

Reply:
Like this ?
```
ifeq ($(CONFIG_DEBUG_CUSTOMOPT),y)
  WCFLAGS  += $(CONFIG_DEBUG_OPTLEVEL)
else ifeq ($(CONFIG_DEBUG_FULLOPT),y)
    WCFLAGS  += -Oz
endif
```

Reply:
Yes, since many native options aren't usable to wasm compiler.